recipes/igwn-lldd-common/meta.yaml

```
@@ -0,0 +1,65 @@
+{% set name = "igwn-lldd-common" %}
+{% set version = "0.1.0" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  url: https://software.igwn.org/lscsoft/source/{{ name }}-{{ version }}.tar.gz
+  sha256: fa70188e6185e807c92fd10f2b82bcd6046435148aebda18fb3a0f8b419a6ef9
+
+build:
+  number: 0
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vvv
+  run_exports:
+    - {{ pin_subpackage("igwn-lldd-common", max_pin="x") }}
+
+requirements:
+  host:
+    - pip
+    - python >=3.6
+    - setuptools
+  run:
+    - python >=3.6
+    - python-confluent-kafka
+    - kafka-python
```

As we discussed privately, I think you only need `python-confluent-kafka` here (to match the Python package), but support for the other kafka implementation should be documented (not here).
```suggestion
    - python-confluent-kafka
```

---

I have discussed this with Patrick and he said that `python-confluent-kafka` is already the case for `RPMs`